stylesheet.css

```diff
@@ -0,0 +1,11 @@
+

```

Comment:
The CSS should be within the `Image.module.css` file `src/components/common/Image/Image.module.css`.
Placed next to the `Image.js` file